src/browser/modules/App/App.jsx

```diff
@@ -136,33 +120,16 @@ export function App (props) {
 
   return (
     <ErrorBoundary>
+      <RelateApi
+        defaultConnectionData={defaultConnectionData}
+        setEnvironmentTheme={setEnvironmentTheme}
+      />
```

Comment:
Personally I dont mind this pattern, but in a way it would be clearer if `RelateApi` had children to clarify dependents. Up to you!